res/css/views/rooms/_EventTile.pcss

```diff
@@ -827,6 +827,9 @@ $left-gutter: 64px;
     }
 }
 
+.mx_Emote {
+    height: 30px;
+}
```

Comment:
Given this is top level and not nested it does not belong in an unrelated styles file, should be in a file specific to `mx_Emote`